src/index.html

```diff
@@ -11,6 +11,15 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form>
+      <div data-qa="big" class="bars bar__big">
+        <input class="bar__inputs bar__input__big" type="text"
+        placeholder="Try “Los Angeles”" >
```

Comment:
Need to fix it everywhere
```suggestion
        <input
          class="bar__inputs bar__input__big"
          type="text"
          placeholder="Try “Los Angeles”"
        >
```